automation/Commands/browser_commands.py

```diff
@@ -227,16 +227,35 @@ def execute(
                 pass
 
 
-def save_screenshot(visit_id, browser_id, driver, manager_params, suffix=""):
-    """ Save a screenshot of the current viewport"""
-    if suffix != "":
-        suffix = "-" + suffix
+class SaveScreenShotCommand(BaseCommand):
+    def execute(
+        self,
+        webdriver,
+        browser_settings,
+        browser_params,
+        manager_params,
+        extension_socket,
+    ):
+        if suffix != "":
+            suffix = "-" + suffix
```

Comment:
```suggestion
        if self.suffix != "":
            self.suffix = "-" + suffix
```
since you don't get passed the suffix as a parameter but instead have to save it at construction time.